doc/source/tune/_tutorials/tune-xgboost.rst

```diff
@@ -0,0 +1,510 @@
+    |---------------------------------+------------+-------+-------------+-------------+--------------------+-------------+----------+--------+------------------|
+    | train_breast_cancer_806ea_00000 | TERMINATED |       | 0.0371055   |           2 |                  1 |    0.611729 | 0.951049 |     11 |        0.339279  |
+    | train_breast_cancer_806ea_00001 | TERMINATED |       | 0.0324613   |           3 |                  2 |    0.643815 |          |      4 |        0.230338  |
+    | train_breast_cancer_806ea_00002 | TERMINATED |       | 0.0100875   |           4 |                  3 |    0.985147 |          |      2 |        0.0661929 |
+    | train_breast_cancer_806ea_00003 | TERMINATED |       | 0.00124263  |           1 |                  3 |    0.890299 |          |      1 |        0.0201721 |
+    | train_breast_cancer_806ea_00004 | TERMINATED |       | 0.000230373 |           5 |                  3 |    0.627611 |          |      1 |        0.0265107 |
+    | train_breast_cancer_806ea_00005 | TERMINATED |       | 0.000186942 |           5 |                  2 |    0.831801 |          |      1 |        0.026082  |
+    | train_breast_cancer_806ea_00006 | TERMINATED |       | 0.00871051  |           2 |                  3 |    0.721523 | 0.958042 |     11 |        0.299392  |
+    | train_breast_cancer_806ea_00007 | TERMINATED |       | 0.00440949  |           2 |                  3 |    0.606252 |          |      1 |        0.0210171 |
+    | train_breast_cancer_806ea_00008 | TERMINATED |       | 0.00948289  |           5 |                  2 |    0.892979 |          |      2 |        0.140424  |
+    | train_breast_cancer_806ea_00009 | TERMINATED |       | 0.0514017   |           2 |                  1 |    0.859864 | 0.972028 |     11 |        0.365437  |
+    +---------------------------------+------------+-------+-------------+-------------+--------------------+-------------+----------+--------+------------------+
+
+As you can see, four trials have been stopped after just one iteration, two after two iterations,
+one after four iterations, and the three most promising configurations have been run for
+ten iterations. The 11 is due to the fact that we finally report the accuracy after
+training the full model, which is internally interpreted as another iteration.
+
+Using fractional GPUs
+---------------------
+You can often accelerate your training by using GPUs in addition to CPUs. However,
+you usually don't have as many GPUs as you have trials to run. For instance, if you
+run 10 Tune trials in parallel, you usually don't have access to 10 separate GPUs.
+
+Tune supports *fractional GPUs*. This means that each task is assigned a fraction
+of the GPU memory for training. For 10 tasks, this could look like this:
+
+.. code-block:: python
+   :emphasize-lines: 3
+
+    tune.run(
+        train_breast_cancer,
+        resources_per_trial={"cpu": 1, "gpu": 0.1},
+        config=config,
```

Comment:
Do you have to do anything with the XGBoost model to actually use the GPU?

Reply:
Right, you have to set `tree_method="gpu_hist"`. Will update it